.github/workflows/upload.yml

```diff
@@ -0,0 +1,28 @@
+name: Upload plugin to PyPi on Github release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  upload:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Build and install Plugin
+        run: |
+          python -m pip install --upgrade pip wheel
+          python setup.py bdist_wheel
+          pip install dist/PennyLane*.whl
```

Comment:
Likely we could even skip this step, as we're not running the test suite (hence the plugin doesn't have to be installed).
```suggestion
```